### PR TITLE
Added dex.ag ids by symbol

### PR DESCRIPTION
**Pull request by blurpesec**
Added support for dex.ag ids

### CODE REVIEW

File: assets/assets.json [hunk 1]
```diff
@@ -3985,5 +3687,303 @@
   },
   "10b38465-640c-509a-b047-8d9671a245ae": {
     "coinGeckoId": "vechain-old-erc20"
+  },
```
Comment (Mrtenz) [inline assets/assets.json]: Do you know why a bunch of stuff is moved to the bottom here?

File: src/assets.ts [hunk 2]
```diff
@@ -94,6 +96,7 @@ export const getParsedAssets = async (): Promise<Record<string, ParsedAsset>> =>
 
   return Object.keys(assets).reduce<Record<string, ParsedAsset>>((object, key) => {
     const asset = assets[key];
+    console.log(asset, isValidAsset(asset));
```
Comment (Mrtenz) [inline src/assets.ts]: Can be removed

File: src/services/dexAG.test.ts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  getInitialCache: jest.fn(async () => CACHE_DATA),
+
+  withCache: jest.requireActual('./cache').withCache
+}));
+
+afterEach(() => {
+  jest.clearAllMocks();
+});
+
+describe('matchCoinGeckoId', () => {
```
Comment (Mrtenz) [inline src/services/dexAG.test.ts]: -> dexAG

File: src/services/dexAG.ts [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+export const matchDexAgId = async (
+  cache: Partial<DexAgData>,
+  asset: Asset
+): Promise<CachedResponse<DexAgData, string | null>> => {
+  const data = cache;
+  if (!data || !data.coins) {
+    data.coins = await getInitialCache<DexAgCoin[]>(DEXAG_API_ENDPOINT);
+  }
+
+  const match = data.coins.find(coin => coin.symbol.toLowerCase() === asset.symbol.toLowerCase());
```
Comment (Mrtenz) [inline src/services/dexAG.ts]: We should probably use the name + symbol here, to prevent false positives?

File: assets/assets.json (hunk 1 above)
Comment (blurpesec) [reply assets/assets.json]: Nope! No idea.

File: src/services/dexAG.ts (hunk 4 above)
Comment (blurpesec) [reply src/services/dexAG.ts]: We want max coverage, with a tolerance of /some/ false positives.

**Pull request merged**